app/app_test.go

```diff
@@ -726,19 +726,38 @@ func TestRolesAdjustment_CantReplaceVoter(t *testing.T) {
 	defer cleanups[0]()
 	defer cleanups[1]()
 
-	// A voter and a spare go offline.
```

Comment:
As commented in https://github.com/canonical/go-dqlite/pull/179, this test tests that an offline voter's role is not transferred to another node if there is no other node to take it's role. This test should still succeed without adaptations.